src/utils/verifyOTP.js

```diff
@@ -0,0 +1,23 @@
+import { Op } from "sequelize";
+import {User,OTP} from "../database/models";
+const verifyOTP=async(req,res)=>{
+    const {email,otp}=req.body
+     const user=await OTP.findOne({
+        where:{
+            otp:otp,
+            createdAt:{
+                [Op.gte]:new Date(new Date()-10*60*1000),
+            },
+        },
+    })
+    if(!user){
+        return res.send("invalid OTP!")
+    }
+    //clear the otp and log the user in
+    else{
+    user.otp=null;
+    await user.save()
+    res.send("logged in successfully")
```

Comment:
Please you are supposed to send token once a user logged in! also remember to include status code and use JSON format